### DISCUSSION TITLE
DBC-Wiki/dbc-introduction/rent-virtual

### DISCUSSION THREAD

**Discussion by giscus** [DeepBrainChain/DBC-Wiki#53] [Announcements]
# DBC-Wiki/dbc-introduction/rent-virtual

一. 注册钱包 进入我的钱包页面 --> 点击 创建钱包账号 按钮，按提示完成操作即可 二. 购买 DBC 租用虚拟机时，需要钱包里面有一定数量的 DBC，1 个 DBC 目前不到 3 分钱，点击 如何购买 DBC，进入购买 DBC 不是积分，他是一种数字货币，他的价格是时刻都在波动的，购买 DBC 后，不仅仅在当前网站可以使用，也可以转账到其他支持 ...

https://deepbrainchain.github.io/DBC-Wiki/dbc-introduction/rent-virtual.html

**Comment by catqaq** [comment]
续租失败怎么办？
